common/models/create_deploy_request.go

```diff
@@ -1,16 +1,27 @@
 package models
 
-import "fmt"
+import (
+	"fmt"
+	"strings"
+)
 
 type CreateDeployRequest struct {
 	DeployName string `json:"deploy_name"`
-	Dockerrun  []byte `json:"dockerrun"`
+	DeployFile []byte `json:"deploy_file"`
 }
 
 func (c CreateDeployRequest) Validate() error {
 	if c.DeployName == "" {
 		return fmt.Errorf("Deploy Name is required")
 	}
 
+	if strings.Contains(c.DeployName, ".") {
```

Comment:
This is ok to have, our new implementation should be able to work around this

Reply:
✅